apollo-router/src/plugins/expose_query_plan.rs

```diff
@@ -218,7 +218,8 @@ mod tests {
             build_mock_supergraph(serde_json::json! {{
                 "plugins": {
                     "experimental.expose_query_plan": true
-                }
+                },
+                "include_subgraph_errors": { "all": true }
```

Comment:
Is this necessary? I imagine it would be if you added the errors tests suggested below, but right now it seems unnecessary.